Dockerfiles/midstream/unit_tests.py

```diff
@@ -3,78 +3,72 @@
 
     # Set env variable to custom made image that makes playbook extract-operator-bundle.yml fail
     # Due to parsing error
     def test_negative_parsing(self):
-        exec_cmd = """
-        export IMAGE_TO_TEST=quay.io/cvpops/test-operator:missing-alm-examples-v1
-        /run_tests.py
-        """
-        result = subprocess.run(exec_cmd, shell=True)
-        with open(".errormessage") as error_file:
+        self.env["IMAGE_TO_TEST"] = "quay.io/cvpops/test-operator:missing-alm-examples-v1"
+        result = subprocess.run(self.exec_cmd,
+                                shell=True,
+                                env=self.env,
+                                cwd=SCRIPT_DIR)
+        with open(SCRIPT_DIR+".errormessage") as error_file:
             self.assertIn("Result code:", error_file.read(), "Result code not found in %s" % error_file)
         self.assertEqual(50, result.returncode)
-        os.remove(".errormessage")
 
     # Set env variable to custom made image that makes validation fails with following error:
     # ERRO[0003] error validating format in /tmp/bundle-688328851: Bundle validation errors: couldn't
     # parse dependency of type olm.crd
     def test_negative_image_bundle_validation(self):
-        exec_cmd = """
-        export IMAGE_TO_TEST=quay.io/cvpops/test-operator:invalid-dependencies-v1
-        /run_tests.py
-        """
-        result = subprocess.run(exec_cmd, shell=True)
-        with open("/project/output/validation-output.txt") as error_file:
+        self.env["IMAGE_TO_TEST"] = "quay.io/cvpops/test-operator:invalid-dependencies-v1"
+        result = subprocess.run(self.exec_cmd,
+                                shell=True,
+                                env=self.env,
+                                cwd=SCRIPT_DIR)
+        with open(SCRIPT_DIR+"/output/validation-output.txt") as error_file:
             self.assertIn("Bundle validation errors: couldn't parse dependency of type olm.crd", error_file.read(), "Result code not found in %s" % error_file)
         self.assertEqual(70, result.returncode)
-        os.remove("/project/output/validation-output.txt")
 
     # Don't set env variable IMAGE_TO_TEST case
     def test_negative_image_to_test_not_set(self):
-        exec_cmd = """
-        /run_tests.py
-        """
-        result = subprocess.run(exec_cmd, shell=True)
-        with open(".errormessage") as error_file:
+        self.env.pop('IMAGE_TO_TEST', None)
+        result = subprocess.run(self.exec_cmd,
+                                shell=True,
+                                env=self.env,
+                                cwd=SCRIPT_DIR)
+        with open(SCRIPT_DIR+".errormessage") as error_file:
             self.assertIn("Result code: 102 Error message: Environment variable IMAGE_TO_TEST not set!", error_file.read(), "Result code not found in %s" % error_file)
         self.assertEqual(102, result.returncode)
-        os.remove(".errormessage")
```

Comment:
Why it was removed? Each test should have clean "env" and leftover from previous test shoudl not cause failure to other test.

Reply:
with the introduction of the logger, .errrormessage acts as a logfile. 
we need not have a check if the .errormessage is created or not. 
I felt we just need to check if the Result cide is inside the log file or not. 
I felt we need not have a new .errormessage file removed.